Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml

```diff
@@ -1327,51 +1497,492 @@ outputs:
+  description: Endpoint user property 1.
+- contextPath: McAfee.ePO.Endpoint.UserProperty2
+  description: Endpoint user property 2.
+- contextPath: McAfee.ePO.Endpoint.UserProperty3
+  description: Endpoint user property 3.
+- contextPath: McAfee.ePO.Endpoint.UserProperty4
+  description: Endpoint user property 4.
+- contextPath: McAfee.ePO.Endpoint.SysvolFreeSpace
+  description: The amount of system volume free space in the endpoint.
+- contextPath: McAfee.ePO.Endpoint.SysvolTotalSpace
+  description: The total amount of system volume space in the endpoint.
+- contextPath: McAfee.ePO.Endpoint.Tags
+  description: Endpoint ePO tags.
+- contextPath: McAfee.ePO.Endpoint.ExcludedTags
+  description: Endpoint EPO excluded tags.
+- contextPath: McAfee.ePO.Endpoint.LastUpdate
+  description: The date the endpoint was last updated.
+- contextPath: McAfee.ePO.Endpoint.ManagedState
+  description: Endpoint managed state.
+- contextPath: McAfee.ePO.Endpoint.AgentGUID
+  description: Endpoint agent GUID.
+- contextPath: McAfee.ePO.Endpoint.AgentVersion
+  description: Endpoint agent version.
+- contextPath: McAfee.ePO.Endpoint.AutoID
+  description: Endpoint auto ID.
+- contextPath: CrowdStrike.Device.ID
+  description: The ID of the device.
+- contextPath: CrowdStrike.Device.LocalIP
+  description: The local IP address of the device.
+- contextPath: CrowdStrike.Device.ExternalIP
+  description: The external IP address of the device.
+- contextPath: CrowdStrike.Device.Hostname
+  description: The host name of the device.
+- contextPath: CrowdStrike.Device.OS
+  description: The operating system of the device.
+- contextPath: CrowdStrike.Device.MacAddress
+  description: The MAC address of the device.
+- contextPath: CrowdStrike.Device.FirstSeen
+  description: The first time the device was seen.
+- contextPath: CrowdStrike.Device.LastSeen
+  description: The last time the device was seen.
+- contextPath: CrowdStrike.Device.PolicyType
+  description: The policy type of the device.
+- contextPath: CrowdStrike.Device.Status
+  description: The device status.
 tests:
-- Field polling test
-- test-domain-indicator
-- Calculate Severity - Generic v2 - Test
-- Wait Until Datetime - Test
-- Calculate Severity - Standard - Test
-- Extract Indicators From File - Generic v2 - Test
-- URL Enrichment - Generic v2 - Test
-- File Enrichment - Generic v2 - Test
-- Domain Enrichment - Generic v2 - Test
-- Account Enrichment - Generic v2.1 - Test
-- IP Enrichment - Generic v2 - Test
-- Get endpoint details - Generic - test
-- Detonate URL - Generic Test
-- Email Address Enrichment - Generic v2.1 - Test
-- Generic Polling Test
-- Send Investigation Summary Reports - Test
-- Detonate File - No Files test
-- Test Convert file hash to corresponding hashes
-- Detonate File - Generic Test
 - Endpoint Enrichment - Generic v2.1 - Test
-- Isolate and unisolate endpoint - test
-- Block IP - Generic V3_Test
 fromversion: 5.0.0
 toversion: 6.7.9
```

Comment:
Should even touch this PB if this is under 6.8?

Reply:
I got this PB and I also need to update the 'Endpoint_Enrichment_-_Generic_v2.1_6_8.yml' playbook